style.css

```diff
@@ -406,3 +536,305 @@ footer > hr {
   background: #0d151e;
   border-radius: 10px;
 }
+
+/* desktop version */
+@media only screen and (min-width: 768px) {
+  .toggle-button {
+    display: none;
+  }
+
+  .navbar-links {
+    width: 100%;
+  }
+
+  .logo {
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .navbar-links ul {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: flex-end;
+  }
+
+  .navbar-links li {
+    text-align: center;
+    row-gap: 1px;
+  }
+
+  .navbar-links li a {
+    padding: 0.5rem 1rem;
+  }
+
+  .initial {
+    display: flex;
+    flex-direction: row-reverse;
+    align-items: center;
+    justify-content: revert;
+    background: url(images/desktop-header-background.png);
+    background-size: 380px;
+    background-repeat: no-repeat;
+    background-position: right top;
+    background-color: #1c1a19;
+  }
+
+  .info-main {
+    width: 85%;
+  }
+
+  .icons {
+    width: 15%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 10px;
+  }
+
+  h1.info {
+    font-size: 54px;
+    width: 41%;
+  }
+
+  p.main-text {
+    width: 60%;
+  }
+
+  .main-container {
+    background-color: white;
+  }
+
+  .works-1 {
+    background: #fff;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 2;
+    border-radius: 0;
+    row-gap: 0;
+    column-gap: 12px;
+    margin-top: 4px;
+    margin-bottom: 24px;
+    margin-left: 30px;
+    margin-right: 0;
+  }
```

Comment:
Great job implementing the desktop version :+1: , The project cards are not how they are in the original figma design, please make sure they are setup correctly, your design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/200121916-f76aa628-771d-408e-bf1b-78c3de3d92cb.png)
Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/200121930-51766c62-e639-40b0-b654-cd226ffef67f.png)
Please try to use flex for individual cards layout and make it as close to the figma design as possible.